test/e2e/02_createClient.js

```diff
@@ -0,0 +1,18 @@
+var conf = require('../../nightwatch.conf.js');
+
+module.exports = {
+    'Create new client': function (browser) {
+        var login = browser.page.login();
```

Comment:
Login code is repeated in every test - should it be moved to a separate function?